ewallet.go

```diff
@@ -12,14 +12,25 @@ const (
 	EWalletTypeLINKAJA EWalletTypeEnum = "LINKAJA"
 )
 
+// EWalletXApiVersionEnum constants are the available e-wallet "X-API-VERSION"s
+type EWalletXApiVersionEnum string
+
+// This consists the values that EWalletXApiVersionEnum can take
+const (
+	EWalletXApiVersionEnum20190204 EWalletXApiVersionEnum = "2019-02-04"
+	EWalletXApiVersionEnum20200201 EWalletXApiVersionEnum = "2020-02-01"
+)
```

Comment:
Let's not have an enum for this as in the ideal case, date-based versioning will be any date within a date range of a version. @slzhffktm Let's accept an arbitrary string for this field

Reply:
got it